acmw-db-app/components/ScholarshipReqForm.jsx

```diff
@@ -33,17 +33,24 @@ const ScholarshipReqForm = (props) => {
         }
     }
 
-    const onSubmit = () => {
+    const onSubmit = async () => {
         if (studentLName.current != "" && reqDesc.current != "") {
             setFormError("")
 
-            // TO DO Update GHC Req here
-
-            console.log(studentFName.current + " " + studentLName.current + " - " + reqType.current + ": " + reqDesc.current)
+            const requestOptions = {
+              method: 'POST',
+              body: JSON.stringify(
+                { name_dot_num: studentLName.current,
+                  student_id: studentId.current,
+                  req_type: reqType.current,
+                  req_desc: reqDesc.current
+                }
+              )
+            };
+            const res = await fetch('/api/ghc/enter-external-scholarship', requestOptions);
             setShowNotif(true)
```

Comment:
Needs to use a subscribed reference which there are examples of in other front end components to prevent updates to unmounted components.